docker/Dockerfile

```diff
@@ -42,19 +39,23 @@ RUN rm /tmp/search-guard-6-6.8.6-25.5.zip
 # Add default roles
 ADD sg_roles_mapping.yml /usr/share/elasticsearch/plugins/search-guard-6/sgconfig/sg_roles_mapping.yml
 
+# Allow read-only anonymous access
+RUN sed -i "s/anonymous_auth_enabled: false/anonymous_auth_enabled: true/" /elasticsearch/plugins/search-guard-6/sgconfig/sg_config.yml
```

Comment:
These two lines are out of the scope of this PR. Should not be added.

Reply:
Apologies, this leaked from some other work I've been doing - I've removed it from this PR.